d2l-button-icon.html

```diff
@@ -2,6 +2,7 @@
 <link rel="import" href="../d2l-colors/d2l-colors.html">
 <link rel="import" href="../d2l-icons/d2l-icons.html">
 <link rel="import" href="../d2l-polymer-behaviors/d2l-focusable-behavior.html">
+<link rel="import" href="../d2l-polymer-behaviors/d2l-visible-on-ancestor-behavior.html">
```

Comment:
I love that we can add this behavior with 3 lines of code, including the import statement.  ;)

Reply:
🎉